services/alert/service.go

```diff
@@ -769,7 +770,15 @@ func (s *Service) createHandlerFromSpec(spec HandlerSpec) (handler, error) {
 		}
 		h = NewPublishHandler(c, s.logger)
 	case "sensu":
-		h = s.SensuService.Handler(s.logger)
+		c := sensu.HandlerConfig{}
+		err = decodeOptions(spec.Options, &c)
+		if err != nil {
+			return handler{}, err
+		}
+		h, err := s.SensuService.Handler(c, s.logger)
```

Comment:
This is where a topic handler calls into the SensuService for the sensu handler. And it looks correct.